lib/elastic_apm/spies/sequel.rb

```diff
@@ -28,7 +28,7 @@ class SequelSpy
       ACTION = 'query'
 
       def self.summarizer
-        @summarizer ||= Sql.summarizer
+        @summarizer ||= Sql::Signature::Summarizer.new
```

Comment:
Do we even need this method now? We could just get rid of the `Sql.summarizer` method.

Reply:
The memoization method? I guess not necessarily.